ui/lib/core/addon/templates/components/ttl-picker.hbs

```diff
@@ -2,18 +2,16 @@
 <MessageError @errorMessage={{errorMessage}} data-test-ttl-error />
 <div class="field is-grouped">
   <div class="control is-expanded">
-    <input data-test-ttl-value value={{time}} id="time-{{elementId}}" type="text" name="time" class="input" oninput={{action 'changedValue' 'time'}}
+    <input data-test-ttl-value value={{time}} id="time-{{elementId}}" type="text" name="time" class="input" oninput={{action (action 'changedValue' 'time') value="target.value"}}
       pattern="[0-9]*" />
   </div>
   <div class="control is-expanded">
```

Comment:
Do we still need this outer div? I see there's `control` call in the Select component, but not sure if `is-expanded` is still necessary.

Reply:
we do indeed - without `control is-expanded`, the select dropdown shrinks to take up only a small width compared to the text input next to it.